docs/source/contributing/using_gitpod.md

```diff
@@ -0,0 +1,126 @@
+(using_gitpod)=
+# Using Gitpod
+
+## About Gitpod
+[Gitpod](https://www.gitpod.io/) is a browser-based development environment.
+
+These are some benefits to using Gitpod:
+
+- Bypass local computer configuration and technical issues
+- Save time by using a pre-configured virtual environment for contributing to open source
+- Save space on your local computer
+
+## Using Gitpod to Contribute to PyMC
+
+These instructions are for contributing specifically to the [pymc-devs/pymc](https://github.com/pymc-devs/pymc) repo.
+
+### Gitpod Workflow
+
+1. Fork the pymc repo: [https://github.com/pymc-devs/pymc](https://github.com/pymc-devs/pymc)
+
+2. Create a Gitpod account. You can login and authorize access via your GitHub account:  [https://gitpod.io/](https://gitpod.io/)
+
+:::{note}
+Gitpod will show up as an authorized application in your GitHub account here: [https://github.com/settings/applications](https://github.com/settings/applications)
+:::
+
+3. Grant GitHub / Gitpod integration permissions.
+
+    a) Go to: [https://gitpod.io/user/integrations](https://gitpod.io/user/integrations)
+
+    b) Select GitHub and then "edit permissions"
+
+    c) Select these permission: user:email, public_repo, repo, workflow
+
+:::{figure-md} Gitpod integration
+
+![gitpod_integration](gitpod/gitpod_integration.png)
+
+Gitpod [integration options](https://gitpod.io/user/integrations)
+:::
+
+4. Within Gitpod, create a ["New Workspace"](https://gitpod.io/workspaces).  Here you will want to select the forked pymc repo. If you don't see it, you can paste into the "Context URL" your forked repo path.  For example:  `https://github.com/reshamas/pymc`.  Then select "New Workspace".
```

Comment:
Given that you have an actual fork under your username, I worry that including this link is too tempting of an invitation for people who aren't paying attention to start a workspace on your fork.

```suggestion
4. Within Gitpod, create a ["New Workspace"](https://gitpod.io/workspaces).  Here you will want to select the forked pymc repo. If you don't see it, you can paste into the "Context URL" your forked repo path.  For example:  `https://github.com/yourusername/pymc`.  Then select "New Workspace".
```